docker/runtime/python-2.7/uwsgi.ini

```diff
@@ -0,0 +1,11 @@
+[uwsgi]
+plugins-dir = /usr/lib/uwsgi
+need-plugin = python
+pythonpath = /usr/local/lib/python2.7
+pythonpath = /usr/local/lib/python2.7/site-packages
```

Comment:
I don't think this path will load the dependencies specified in the requirements.txt file.